kalibro_client/processor/module_result.py

```diff
@@ -2,8 +2,10 @@
     entity_name_decorator
 from kalibro_client.processor.base import Base
 from kalibro_client.processor import KalibroModule, Processing, MetricResult, \
-    Repository
+    Repository, TreeMetricResult
 import kalibro_client.miscellaneous.date_module_result
+from kalibro_client.processor.hotspot_metric_result import HotspotMetricResult
```

Comment:
When we first saw the line that was already there we though the same thing, but changing it led to some circular import issues. I'll admit I don't actually understand them so well, so I'll read about them a bit and see if there's a way to remove the absolute import here.